packages/wonder-blocks-modal/components/modal-header.test.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import React from "react";
+import {shallow} from "enzyme";
+
+import expectRenderError from "../../../utils/testing/expect-render-error.js";
+import ModalHeader from "./modal-header.js";
+
+describe("ModalHeader", () => {
+    test("using only `subtitle` should render the header", () => {
+        const wrapper = shallow(
+            <ModalHeader title="Title" subtitle="Subtitle" />,
+        );
+
+        // we make sure the component is rendered
```

Comment:
I used this approach b/c we are already generating snapshots automatically. Or, do you think it would be better to use `toMatchSnapshot()` instead?

Reply:
I think this is fine for now.  We talked a while back about changing the generated snapshots to match shallow snapshots at which point we'd probably want to snapshot tests to this (and other test suites) that compare full snapshots.